independent-projects/bootstrap/pom.xml

```diff
@@ -71,7 +72,10 @@
         <slf4j-api.version>1.7.36</slf4j-api.version>
         <graal-sdk.version>22.3.0</graal-sdk.version>
         <plexus-classworlds.version>2.6.0</plexus-classworlds.version> <!-- not actually used but ClassRealm class is referenced from the API used in BootstrapWagonConfigurator -->
+        <plexus-cipher.version>1.7</plexus-cipher.version>
+        <plexus-utils.version>3.5.0</plexus-utils.version>
```

Comment:
Plexus-utils should be bumped to 3.5.1 at the same time.

Reply:
Thanks, updating.